hazelcast-hibernate5/src/main/java/com/hazelcast/hibernate/CacheEnvironment.java

```diff
@@ -53,12 +53,7 @@
     /**
      * Property to configure Hazelcast client cluster name
      */
-    public static final String NATIVE_CLIENT_CLUSTER = "hibernate.cache.hazelcast.native_client_group";
-
-    /**
-     * Property to configure Hazelcast client group password
-     */
-    public static final String NATIVE_CLIENT_PASSWORD = "hibernate.cache.hazelcast.native_client_password";
+    public static final String NATIVE_CLIENT_CLUSTER = "hibernate.cache.hazelcast.native_client_cluster";
```

Comment:
I would change the name to `NATIVE_CLIENT_CLUSTER_NAME = "hibernate.cache.hazelcast.native_client_cluster_name";` if you have no objections.

Reply:
Changed on a579345